### ISSUE TITLE
/proc/<pid> prefixed to rootfs path

### ISSUE THREAD

**Issue by crosbymichael** [NVIDIA/libnvidia-container#19]
I'm running this in supervised mode with the following args and get the error where the rootfs is not found because the cli tool is prefixing the rootfs paths with /proc/<pid>.

I cannot figureout what is wrong.

```
# args 
[--load-kmods configure --device=0 --utility --pid=2087 /run/containerds/gpus/rootfs]
#error
ERRO[0002] exit status 1: nvidia-container-cli: container error: open failed: /proc/2087/root/run/containers/gpus/rootfs: no such file or directory
```

**Comment by 3XX0** [comment]
Yes, the CLI needs to be called before the `pivot_root` (see [example](https://github.com/NVIDIA/libnvidia-container#command-line-example)), thus `procfs` needs to be mounted and should be able to access the PID specified by `--pid` (i.e. be in an ancestor PID namespace)

**Comment by crosbymichael** [comment]
Thanks for confirming, sorry for the noise.

**Comment by 3XX0** [comment]
If it helps the setup needs to be similar to a prestart runc hook. This is how we currently implement it in our [runtime hook](https://github.com/NVIDIA/nvidia-container-runtime)
